### ISSUE TITLE
License

### ISSUE THREAD

**Issue by stweil** [kitodo/kitodo-production#1301]
It is currently not easy to find the license used by Kitodo.Production as a whole. Ideally that information should be supplied in a GitHub compatible form, so GitHub would display it. Adding it to the README would also be fine.

In addition it would be good to have also license information for all images. Can some of those images be used for Wikipedia, for example?

**Comment by stweil** [comment]
Is the license for the image files now clear? Can they be used on Wikipedia? If they also use GPL3 and not allow other licenses, that would not be possible.

**Comment by Kathrin-Huber** [comment]
What image files do you referre to?

**Comment by stweil** [comment]
The most important one is the Kitodo logo which would be useful for the [Wikipedia article on Kitodo](https://en.wikipedia.org/wiki/Kitodo). That requires an open license which is [compatible with Wiki Commons](https://commons.wikimedia.org/wiki/Commons:Licensing).

Other [Kitodo images for Commons](https://commons.wikimedia.org/wiki/Category:Kitodo) would also be appreciated.

**Comment by Kathrin-Huber** [comment]
I think it is a question  you should discuss with the Kitodo Board, isn't it?

**Comment by Erikmitk** [comment]
@stweil The logo itself is provided on the kitodo.org homepage (https://www.kitodo.org/community/logo-und-praesentationen/) as well for free. But you're right. There's also no explicit licence information available. So I'd too suggest to point this question not at the developers here but to the Geschäftsstelle of the Kitodo e.V.

**Comment by stweil** [comment]
I wrote to the Geschäftsstelle now. As far as I see we have two relevant logos: the [key symbol](https://s3.amazonaws.com/cloud.ohloh.net/attachments/91091/kitodo_med.png) and the [Verein's logo](https://www.kitodo.org/fileadmin/groups/kitodo/Logo/kitodo_logo_rgb.png). For Wikimedia Commons a CC license would be best, if possible with each logo in SVG format.

**Comment by Kathrin-Huber** [comment]
Not a code problem

**Comment by stweil** [comment]
... but still a problem of this repository which not only contains code but also images and other non-code files.

And for the images the GPL license is a real problem because such images cannot be used in Wikipedia. Even screenshots in scientific articles or presentations might be problematic.

**Comment by stweil** [comment]
> I think it is a question you should discuss with the Kitodo Board, isn't it?

Right, so let's ask the @kitodo/kitodo-board. I have these two questions:

1. Kitodo.Production uses the GPL-3.0, but the repository also includes images which ideally should also allow a different license (for example CC-BY 4.0) which allows using these images for example on Wikipedia or in presentations (which are often also published under CC-BY 4.0). **Can we add CC-BY 4.0 as additional license for selected images and icons in this repository?**
2. The fact that the code uses GPL-3.0 is shown in header comments in all relevant source code files. The current template used for these comments is a non-standard text which causes problems (duplicate license files, two licenses shown on GitHub) because it refers to a file GPL-3-License.txt which is also non-standard (see [example of code](https://github.com/kitodo/kitodo-production/blob/main/Kitodo-API/src/main/java/org/kitodo/api/command/CommandInterface.java)). **Can we replace the non-standard header comments by different ones without the problematic reference?**

Current header comment:
``` java
/*
 * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
 *
 * This file is part of the Kitodo project.
 *
 * It is licensed under GNU General Public License version 3 or later.
 *
 * For the full copyright and license information, please read the
 * GPL3-License.txt file that was distributed with this source code.
 */
```

Suggested header comment:
```java
/*
 * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
 *
 * This file is part of the Kitodo project.
 *
 * It is licensed under GNU General Public License version 3 or later.
 */
```

Alternative header comment:
```java
/*
 * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
 *
 * This file is part of the Kitodo project.
 *
 * It is licensed under GNU General Public License version 3 or later.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
```

Alternative minimum header which uses a standard license identifier and could also be added to the other variants (cf. Linux code):
```
/* SPDX-License-Identifier: GPL-3.0-or-later */
```

Header comment which follows the "official" template (see https://www.gnu.org/licenses/gpl-3.0):
```java
/*
 * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
 *
 * This file is part of the Kitodo project.
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
```

**Comment by stweil** [comment]
> The fact that the code uses GPL-3.0 is shown in header comments in all relevant source code files. The current template used for these comments is a non-standard text which causes problems (duplicate license files, two licenses shown on GitHub) because it refers to a file GPL-3-License.txt which is also non-standard (see [example of code](https://github.com/kitodo/kitodo-production/blob/main/Kitodo-API/src/main/java/org/kitodo/api/command/CommandInterface.java)). **Can we replace the non-standard header comments by different ones without the problematic reference?**

@kitodo/kitodo-board, my cited comment is still not answered. New pull requests which add new files still use the non-standard header with the problematic reference to GPL3-License.txt. I suggested several alternatives which would improve the situation and also said that my personal preference is option B (update for two lines, see details above). How can we make progress in this discussion?
